cl/alerts/management/commands/cl_send_alerts.py

```diff
@@ -63,10 +65,30 @@ def send_alert(user_profile, hits):
     txt_template = loader.get_template("alert_email.txt")
     html_template = loader.get_template("alert_email.html")
     context = {"hits": hits}
+    headers = {}
+    query_string = ""
+    if len(hits) == 1:
+        alert = hits[0][0]
+        unsubscribe_path = reverse(
+            "one_click_disable_alert", args=[alert.secret_key]
+        )
+        headers["List-Unsubscribe-Post"] = "List-Unsubscribe=One-Click"
+    else:
+        params = {"keys": [hit[0].secret_key for hit in hits]}
+        query_string = urlencode(params, doseq=True)
+        unsubscribe_path = reverse("disable_alert_list")
+    headers["List-Unsubscribe"] = (
+        f"<https://www.courtlistener.com{unsubscribe_path}?{query_string}>"
```

Comment:
This line is adding an unnecessary `?` when there is only one search alert in the email. I think it doesn't affect the request, but we could just remove it.

```
List-Unsubscribe-Post: List-Unsubscribe=One-Click
List-Unsubscribe: <https://www.courtlistener.com/alert/one-click-disable/secret_key/?>
```

Reply:
Thanks for the suggestion. I updated the f-string to avoid adding the question mark when there isn't a query string.